src/show/TranscriptsCanvas.js

```diff
@@ -429,7 +429,7 @@ const Transcripts = ({keepOnlyTheseColumnIds, columnHeaders, rows, display, conf
                                   xAxisCategories={xAxisCategories}
                                   rows={rows}/>
         <Legend legendItems={Object.keys(DATA_SERIES).map((key) => DATA_SERIES[key]).filter(o => o.name && o.colour)}
-                title={`Dominant: dominant in all samples. Ambiguous: dominant in some samples. Present: non-dominant in all samples`}
+                title={`Dominant: dominant in all samples.\nAmbiguous: dominant in some samples.\nNon-dominant: non-dominant in all samples.`}
```

Comment:
[I didn’t know `\n` was parsed in tooltips!](https://media3.giphy.com/media/3og0IMJcSI8p6hYQXS/giphy.gif)